components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -384,33 +406,22 @@
           }).get();
         },
 
-        attachManagingListeners: function (baseUrl)
+        attachStateChangeCallbacks: function(callback)
         {
           var _this = this;
           var consentList = $(".consents");
           consentList.on("change", "li input", function (event)
           {
             var checkbox = $(event.target);
-            var rest = _this.interactWithManagingRest(checkbox.attr("data-bind"), baseUrl);
+            var name = checkbox.attr("data-bind");
             if (checkbox.is(":checked")) {
-              rest("grant");
+              callback(name, true);
             } else {
-              rest("revoke");
+              callback(name, false);
```

Comment:
Good catch re: jquery. It was used before, I removed that piece of code and did not realize jquery is no longer needed